src/components/cc-zone-picker/cc-zone-picker.js

```diff
@@ -35,13 +38,31 @@ export class CcZonePicker extends CcFormControlElement {
   constructor() {
     super();
 
-    /** @type {ZonesSections} array of zones sections */
-    this.zonesSections = [];
+    /** @type {boolean} whether the whole form should be disabled */
+    this.disabled = false;
+
+    /** @type {boolean} whether the form should be in readonly */
```

Comment:
**suggestion:** let's not mention `form` here because this component is not a `form` by itself.
Maybe something like:
```suggestion
    /** @type {boolean} whether all the form controls should be disabled */
    this.disabled = false;

    /** @type {boolean} whether all the form controls should be readonly */
```